src/main/java/com/walking/intensive/chapter2/task6/Task6.java

```diff
@@ -19,8 +22,13 @@ public static void main(String[] args) {
      * <p>Если входные данные некорректны - метод должен возвращать -1.
      */
     static int getLcm(int m, int n) {
-        // Ваш код
-        return 0;
+        if (m <= 0 || n <=0) {
```

Comment:
Error: eckstyle] [ERROR] /home/runner/work/intensive-tasks-2024/intensive-tasks-2024/src/main/java/com/walking/intensive/chapter2/task6/Task6.java:25:25: '<=' is not followed by whitespace. [WhitespaceAround]